test/evp_test.c

```diff
@@ -2214,28 +2214,28 @@ static int encapsulate(EVP_TEST *t, EVP_PKEY_CTX *ctx, const char *op,
     KEM_DATA *kdata = t->data;
     unsigned char *wrapped = NULL, *secret = NULL;
     size_t wrappedlen = 0, secretlen = 0;
-    OSSL_PARAM params[10] = { OSSL_PARAM_END };
+    OSSL_PARAM params[10];
```

Comment:
I almost feel like there should be a `max_params` variable initialized to `OSSL_NELEM(params)`, with `-1` for the "end" and then another `-1` if `kdata->entropy != NULL`, rather than having a somewhat magic number `OSSL_NELEM(params)-2` in multiple locations.

Reply:
I've done this.